ccpp/data/GFS_typedefs.F90

```diff
@@ -4174,7 +4174,12 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
     Model%ivegsrc          = ivegsrc
     Model%isot             = isot
     Model%use_ufo          = use_ufo
-    Model%vrbliceden_noah  = vrbliceden_noah
+    Model%exticeden        = exticeden
+    if (Model%exticeden .and. (imp_physics /= imp_physics_gfdl .and. imp_physics /= imp_physics_thompson .and. imp_physics /= imp_physics_nssl )) then
```

Comment:
@ericaligo-NOAA I put this check in here because the calculation of rhonewsn in GFS_MP_generic_post is within an IF statement for these 3 MP schemes. If we don't have a check like this, it would be possible to specify `exticeden = T` in the namelist for a suite using a different microphysics and the LSMs would bypass their internal calculation of frozen precipitation density in favor of a bogus/initialized-only value of rhonewsn.